src/pages/registration.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Registration and Invitations
+sub: General registrations is currently on hold
+---
+
+We're so grateful for the enthusiasm we've received,
+and we're enjoying getting to know new faces
+(as well as connect with some old ones).
+While we’re excited to be part of
+a larger extended ‘fediverse’ community,
+we need to keep this server at a size
+that we’re able to afford, maintain, and moderate.
+We think that number is somewhere under 350 members.
+
+As the interest (and membership)
+has grown much faster than expected,
+we've had to pause both member-invites and registrations for now.
+We're managing this space in our spare time,
+and know that we're not ready for the rapid growth
+that many other servers have experienced.
+We don't intend to be an exclusive club,
+but we also aren't set up to be
+a central hub for all things front-end.
+We hope that others will start similar spaces,
+and we can build our community as a federated network,
+rather than centralizing on one domain.
+
+It has also been clear from the early registrations
+that leaving the process unchecked
+would result in a user base that's overwhelmingly cis, white, and male.
+Since we paused registration,
+the same pattern has played out in
+our DMs and emails --
+with primarily white men expecting special invites
+(and getting hostile when denied).
+We believe it's important
+to push against that trend.
+We want to work actively toward developing
+a community that's diverse and inviting
+to everyone who shares these values --
+and we want your help.
+
+### Current Members Can Request Invites
+
+If you have friends or co-workers
+that you think might want to join our community,
+_particularly those that are a different gender or ethnicity than your own_,
+message [@mods](https://front-end.social/@mods) via DM
+and we will see about getting some invites out.
+
+### General Registration Process
+
+We are working to improve the application process
+with a few more questions
```

Comment:
Is this bit still relevant?

Reply:
I think we can probably cut that line and keep the sentence.

Reply:
Maybe just keep the first phrase up to this point.